src/Raven.Client/Documents/Indexes/IndexUpdateClusterState.cs

```diff
@@ -1,19 +1,22 @@
 ﻿namespace Raven.Client.Documents.Indexes
 {
-    internal class ClusterState
+    public class IndexUpdateClusterState
```

Comment:
can we stay with internal?
can we rename to IndexDefinitionClusterState?

Reply:
done